ddtrace/tracer.py

```diff
@@ -795,8 +800,8 @@ def _log_compat(self, level, msg):
         else:
             log.log(level, msg)
 
-    def trace(self, name, service=None, resource=None, span_type=None, span_api=SPAN_API_DATADOG):
-        # type: (str, Optional[str], Optional[str], Optional[str], str) -> Span
+    def trace(self, name, service=None, resource=None, span_type=None, span_api=SPAN_API_DATADOG, span_links=None):
```

Comment:
are these `span_links` argument additions part of the specification for changes to the public API?

Reply:
the docs and tests provided here make it seem like only `span.link_span` is used?

Reply:
Yup. This is part of the spec. We need a way to initialize a span with links. I will add an example when I make `ddtrace.tracing._span_link.SpanLink` public. I am keeping this module private for now because this feature is not supported by the agent or the backend. Full support will likely be released after reinvent.